js/plugins/google-cloud/src/gcpOpenTelemetry.ts

```diff
@@ -338,6 +339,13 @@ class AdjustingTraceExporter implements SpanExporter {
     }
     return span;
   }
+
+  private markGenkitModel(span: ReadableSpan): ReadableSpan {
+    if (span.attributes['genkit:metadata:subtype'] === 'model') {
```

Comment:
In the other examples, I added `&& attributes['genkit:name']` to the condition, just to avoid the corner case where we write a null key. Shouldn't ever happen but...

Reply:
Good call!